tests/e2e/creation_test.go

```diff
@@ -295,7 +305,7 @@ func (tc *testContext) testApplicationCreation(component components.ComponentInt
 	err := wait.PollUntilContextTimeout(tc.ctx, tc.resourceRetryInterval, tc.resourceCreationTimeout, false, func(ctx context.Context) (bool, error) {
 		// TODO: see if checking deployment is a good test, CF does not create deployment
 		appList, err := tc.kubeClient.AppsV1().Deployments(tc.applicationsNamespace).List(ctx, metav1.ListOptions{
-			LabelSelector: odhLabelPrefix + component.GetComponentName(),
+			LabelSelector: labels.ODHAppPrefix + component.GetComponentName(),
```

Comment:
it does not contain final /

Reply:
probably it's supposed to be used as labels.ODH.Component(component.GetComponenName())